### PR TITLE
Use "cldr-core" data to determine default country for languages

### LINKED ISSUE #7: Use Unicode CLDR defaults
https://github.com/unicode-cldr/cldr-core/blob/master/supplemental/likelySubtags.json#L987 for example shows that `pt` actually maps to `pt-Latn-BR` by default and that `pt-PT` is the special case. 

http://cldr.unicode.org/translation/default-content gives an explanation for this:

> The region is typically the one with the most speakers of that language.

I'm wondering if we should adjust to that rule and possibly even use https://github.com/unicode-cldr/cldr-core/blob/master/supplemental/likelySubtags.json as the basis for the region defaults.

### PR DESCRIPTION
**Pull request by Turbo87**
Resolves #7

### CODE REVIEW

**Comment by Turbo87** [comment]
@10xjs any comments on this?

**Comment by 10xjs** [comment]
Looks good to me!

**Comment by 10xjs** [comment]
One nit: update the `README` to reflect the default `en` change 
image

File: index.js [hunk 1]
```diff
@@ -1,175 +1,23 @@
...
+    }
+
+    code = likelySubtag.split(/-|_/).pop();
+    if (!/^[A-Z]{2}$/.test(code)) {
+      return '';
+    }
   }
 
-  if (!code) {
+  if (!code || code === 'ZZ') {
```
Comment (10xjs) [inline index.js]: @Turbo87 Can you comment on the `ZZ` exception?

**Comment by 10xjs** [comment]
@Turbo87 We should avoid adding `require` call. I want to keep this browser compatible. Either we can inline all of the data (and remove the `cldr-core` dependency) or add a build phase to create a universal module.

**Comment by Turbo87** [comment]
ah, right. I wasn't aware that this was supposed to run in the browser too, but now that you say it that totally makes sense. I'll update the PR in the next days to reflect that.

**Comment by Turbo87** [comment]
what I don't understand though: we're using `module.exports = ...` in `index.js`, but that doesn't work in the browser 🤔

**Comment by 10xjs** [comment]
Yeah good point. This won't work unless it's bundled through webpack or similar.

However, directly requiring a `json` file is one step further that will require users to have a loader configured.

Let's merge this as is and then I can add a UMD build step for the entire package in a followup PR.

**Comment by Turbo87** [comment]
> Let's merge this as is

nah, I think I got a better solution for this that will make it a bit easier. just haven't had time yet to implement it...

**Comment by Turbo87** [comment]
@10xjs I've updated the PR now. Could you have a look?

**Comment by 10xjs** [comment]
@Turbo87 LGTM

**Comment by Turbo87** [comment]
@10xjs let me know if you need help with maintenance and releasing here :)

**Comment by 10xjs** [comment]
@Turbo87 Your help is definitely welcome! I'll publish these upgrades now as `0.3.0` and I'll add you on as a repo collaborator.

**Comment by 10xjs** [comment]
`+ locale-emoji@0.3.0`

**Pull request merged**